packages/react-components/src/molecules/PeopleCard.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```

Comment:
Can we move this to `organisms`? Unless you've thought about it and disagree that it fits in there